.github/workflows/main.yml

```diff
@@ -844,7 +844,10 @@ jobs:
       run: |
         echo -e "[ODBC]\nTrace = yes\nTraceFile = /tmp/odbctrace\n\n[DuckDB Driver]\nDriver = "`pwd`"/build/debug/tools/odbc/libduckdb_odbc.so" > odbcinst.ini
         echo -e "[DuckDB]\nDriver = DuckDB Driver\nDatabase=:memory:\n" > odbc.ini
-        (cd psqlodbc && ODBCSYSINI=.. PSQLODBC_TEST_DSN="DuckDB" ./build/debug/psql_odbc_test connect-test) || (cat /tmp/odbctrace; exit 1)
+        cd psqlodbc && export ODBCSYSINI=.. && export PSQLODBC_TEST_DSN="DuckDB" && export ODBC_TRACE=/tmp/odbctrace
+        (./build/debug/psql_odbc_test connect-test && printf "" > $ODBC_TRACE) || (cat $ODBC_TRACE; exit 1)
```

Comment:
You can enable multiple tests in one command line using the catch syntax instead of calling twice.

Reply:
To expand on that since it's not obvious, you can use echo to create a file that contains the tests you want to run (one test name per line) and then use the `-f` parameter of catch to run the tests, e.g.:

```sh
echo "connect-test\nselect-test\n" > supported_tests
(./build/debug/psql_odbc_test -f supported_tests) || (cat $ODBC_TRACE; exit 1)
```

Reply:
Thanks! I've try to look for something like the "-f" argument, but I didn't reach it.
It's good to know this option.